libraries/lambda_handlers/page_view_count_handler.py

```diff
@@ -18,11 +18,7 @@ def _handle(self, event, context):
             data = event['data']
         if 'body-json' in event and isinstance(event['body-json'], dict):
             data.update(event['body-json'])
-        # Set required env_vars
-        env_vars = {
-            'manifest_table_name': self.retrieve(event['vars'], 'manifest_table_name', 'Environment Vars')
-        }
-
+        env_vars = {}
```

Comment:
need to at least pass in the table name, and probably more items.

Reply:
Seems that you should still use staging variables so that you can select between test, dev, and prod tx-manifest tables.

Reply:
The table names can be the same now that each database is different for prod, dev and test.

Reply:
It still uses stage variables...see handler.py and app.py....automatically populates itself from Stage vars. Yet really we don't need to have different names for the tables within a database now.